src/test/scala/com/microsoft/hyperspace/index/IndexManagerTest.scala

```diff
@@ -342,7 +341,7 @@ class IndexManagerTest extends HyperspaceSuite with SQLHelper {
     // 3. Call optimize. Check the metadata. It should not contain small index files created
     // during refresh operations.
     withTempPathAsString { testPath =>
-      withSQLConf(OPTIMIZE_FILE_SIZE_THRESHOLD -> "900") {
+      withSQLConf(OPTIMIZE_FILE_SIZE_THRESHOLD -> "910") {
```

Comment:
is this change necessary?

Reply:
It happens that the small files are between 900 and 910 bytes in Spark 3 (it was ~890 bytes in Spark 2.4). Large files are like ~950 bytes. This seems arbitrary and there should be a better way, but let's keep it this way in this PR.